changelog.txt

```diff
@@ -1,3 +1,18 @@
+4.2.0
+=============
+New Features:
+- **Allow users to specify their own instance metadata**.  For details see https://aka.ms/msal-net-custom-instance-metadata [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1259)
+- **AcquireTokenSilent should not make calls to the network** [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1144)
+- **Improve CA Error Handling** [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1148)
+- **Supportability: MsalException message should contain the framework and version for easier debugging** [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1245)
+- **AcquireTokenSilent access the cache too many times** [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1230)
+- **Allow injecting the Parent Activity/Window in the Client Builder** [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1095)
+- **Add framework and version to MsalException ToString()** [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1260)
+
+Bug Fixes:
+- **Key not valid for use in specified state - happens in .Net Framework 4.7.2 only, not on .Net Core 2.2** [More information on this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1201)
```

Comment:
should say, "resolved the "Key not valid for use in specified state" error when a certificate with a non exportable key is used on .Net Framework 4.7.2+"